### DISCUSSION TITLE
Can't find "example data"

### DISCUSSION THREAD

**Discussion by allseven** [nathanmarlor/foxess_modbus_charge_period_card#20] [Support] [answered]
I'm trying to write an automation using the card but am having difficulties. There is a message included with the card:
"_Charge periods to set. This should be an array of objects, one for each charge period. All charge periods must be specified. See the example data_" but I can't find it. Is it hiding?

**Comment by canton7** [comment, marked as answer]
Are you sure you're looking at the charge period card? That sounds like the help message for the "set charge periods" service provided by the foxess_modbus integration (nothing to do with the card).

Click "GO TO YAML MODE":

![image](https://github.com/nathanmarlor/foxess_modbus_charge_period_card/assets/568104/e2905588-f2dd-4825-b1ca-9f6ad54f1126)

Then:

![image](https://github.com/nathanmarlor/foxess_modbus_charge_period_card/assets/568104/da0f1054-863a-4f07-b186-c55ee8cdcb64)

And "FILL EXAMPLE DATA"

**Reply by allseven** [reply]
Thanks, and apologies for the wrong reference, it was indeed "set charge periods". I've followed your example and it's worked well.
